src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Apprenticeships/ApprenticeshipsDataManagementController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core;
+using Dfc.CourseDirectory.Core.DataManagement.Schemas;
+using Dfc.CourseDirectory.Core.Models;
+using Dfc.CourseDirectory.WebV2.Filters;
+using Dfc.CourseDirectory.WebV2.ModelBinding;
+using Dfc.CourseDirectory.WebV2.Mvc;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Dfc.CourseDirectory.WebV2.Features.DataManagement.Apprenticeships
+{
+    [Route("data-upload/apprenticeships")]
+    [RequireFeatureFlag(FeatureFlags.CoursesDataManagement)]
```

Comment:
Needs its own feature flag
```suggestion
    [RequireFeatureFlag(FeatureFlags.ApprenticeshipsDataManagement)]
```

Reply:
done